### DISCUSSION TITLE
Temperatur-Offset

### DISCUSSION THREAD

**Discussion by Mephisto090** [OpenCCU/OpenCCU#2562] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

wie darf ich diese Hilfe verstehen:
![image](https://github.com/jens-maus/RaspberryMatic/assets/85995176/21f3d692-6b5a-46e2-94d0-6b8b97def457)

Am Heizkörper: 20°C
Am anderen Ende vom Raum: 18°C

Temperatur Offset: -2°C oder +2°C?

ggf. sollte die Hilfe etwas genauer geschrieben werden.

VG

**Comment by Baxxy13** [comment, marked as answer]
-2°C , steht doch auch so im Hilfetext.
Die Hilfetexte kommen direkt von eQ-3, also Verbesserungsvorschläge am besten dorthin richten.
